eo-maven-plugin/src/main/java/org/eolang/maven/GmiMojo.java

```diff
@@ -203,16 +205,12 @@ public final class GmiMojo extends SafeMojo {
      * @implNote {@code property} attribute is omitted for collection
      *  properties since there is no way of passing it via command line.
      * @checkstyle MemberNameCheck (15 lines)
-     * @todo #1146:30min At the moment we don't support pattern matching, but
-     *  double-star means "everything". Let's implement proper matching,
-     *  where "org.eolang.int" would be matched by "org.*.int" and by
-     *  "org.**". The same is true about gmiExclude, let's fix it too.
      */
     @Parameter
-    private Set<String> gmiIncludes = new SetOf<>("**");
+    private Set<String> gmiIncludes = new SetOf<>("[A-Za-z0-9.]+?");
```

Comment:
@OlesiaSub `**` would be more logical to put here, I believe

Reply:
In my implementation ** matches only expressions containing a dot. Beacuse I though you didn't want org.**.foo to match expressions like org.bar.foo, where it's only one word between the dots. Or did I misunderstand you and it should match anything?

Reply:
@yegor256 see above

Reply:
@OlesiaSub `**` means anything _including_ the dot, while `*` means anything _excluding_ the dot